consul/structs/structs.go

```diff
@@ -661,40 +660,10 @@ func (d *DirEntry) Clone() *DirEntry {
 
 type DirEntries []*DirEntry
 
-type KVSOp string
-
-const (
-	KVSSet        KVSOp = "set"
-	KVSDelete           = "delete"
-	KVSDeleteCAS        = "delete-cas" // Delete with check-and-set
-	KVSDeleteTree       = "delete-tree"
-	KVSCAS              = "cas"    // Check-and-set
-	KVSLock             = "lock"   // Lock a key
-	KVSUnlock           = "unlock" // Unlock a key
-
-	// The following operations are only available inside of atomic
-	// transactions via the Txn request.
-	KVSGet          = "get"           // Read the key during the transaction.
-	KVSGetTree      = "get-tree"      // Read all keys with the given prefix during the transaction.
-	KVSCheckSession = "check-session" // Check the session holds the key.
-	KVSCheckIndex   = "check-index"   // Check the modify index of the key.
-)
-
-// IsWrite returns true if the given operation alters the state store.
-func (op KVSOp) IsWrite() bool {
```

Comment:
I see what you did - i'd keep it `IsWrite` but list the write ops here. We also need the new op listed.